projectName-web/src/main/webapp/WEB-INF/views/welcome/home.jsp

```diff
@@ -1,4 +1,4 @@
 <div id="wrapper">
-    <h1>Hello world!</h1>
+    <h1 id="main-content">Hello world!</h1>
```

Comment:
This is not 'main content'.
Something like "title" or "header-title" would be better.